rollup.config.js

```diff
@@ -4,12 +4,13 @@ import typescript from "@rollup/plugin-typescript";
 import dts from "rollup-plugin-dts";
 import postcss from "rollup-plugin-postcss";
 import image from '@rollup/plugin-image';
+import json from '@rollup/plugin-json'
```

Comment:
What's this for?  I don't see any JSON includes or exports here

Reply:
markdown-it includes a json file and the rollup build can't handle packing json files into the bundle without this.

Reply:
Got it, makes sense